site/docs/blogs.md

```diff
@@ -77,26 +119,31 @@ Here is a list of company blogs that talk about Iceberg. The blogs are ordered f
 
 **Authors**: [Ayush Saxena](https://www.linkedin.com/in/ayush151/)
 
+<!-- markdown-link-check-disable-next-line -->
```

Comment:
can you rebase the PR please? This change is already on main

Reply:
should be rebased now